src/components/shared/headers/AppHeader.js

```diff
@@ -10,7 +10,7 @@ class AppHeader extends Component {
   headerLinkClasses(route) {
     var classes = ['header-nav-link'];
     
-    if (this.props.location.pathname == route) {
+    if (this.props.location.pathname === route) {
```

Comment:
My real goal here is to keep the build free of warnings. My reasoning is that if we get used to ignoring warnings then we'll miss new warnings, and at that point it would be better to have no warnings at all. But I think warnings are valuable (especially since eslint knows a whole lot more about js than me lol)

Also I agree simple code inspection is enough to convince me that these are strings, but since we have no tests or typechecker enforcing that, I see no reason we should think that will be true in the future. I'd be more comfortable using === everywhere.